zipline/testing/core.py

```diff
@@ -492,20 +490,17 @@ def create_data_portal(env, tempdir, sim_params, sids, adjustment_reader=None):
         )
 
 
-def write_bcolz_minute_data(env, days, path, df_dict):
+def write_bcolz_minute_data(env, days, path, data):
```

Comment:
correct. I am working on a feature where I am passing all of our writers to a user defined function and it makes it easier to understand the API if we minimize the differences between the minute and daily writers